src/acceptance/acceptance_suite_test.go

```diff
@@ -0,0 +1,88 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestAcceptance(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Acceptance Suite")
+}
+
+var (
+	dora        = "dora"
+	doraWindows = "dora-windows"
+)
+var _ = SynchronizedBeforeSuite(func() {
+	orgName := "orgNameIprotsiuk"
+	spaceName := "spaceNameIprotsiuk"
+
+	CreateOrg(orgName)
+	DeferCleanup(func() {
+		DeleteOrg(orgName)
+	})
+	CreateSpace(orgName, spaceName)
+	DeferCleanup(func() {
+		DeleteSpace(spaceName)
+	})
+
+	targetCmd := exec.Command("cf", "target", "-o", orgName, "-s", spaceName)
+	targetCmd.Stdout = GinkgoWriter
+	targetCmd.Stderr = GinkgoWriter
+
+	Eventually(targetCmd.Run, 60*time.Second, 5*time.Second).Should(Succeed())
+	PushApp(dora)
+	PushApp(doraWindows)
+}, func() {})
+
+func CreateOrg(orgName string) {
+	var cmd *exec.Cmd
+	cmd = exec.Command("cf", "create-org", orgName)
+	cmd.Stdout = GinkgoWriter
+	cmd.Stderr = GinkgoWriter
+	_ = cmd.Run()
```

Comment:
Can you add retries and error checking please? Probably not worth proceeding if we can't create the org.